docs/modules/develop/pages/classes/commands.adoc

```diff
@@ -0,0 +1,84 @@
+= Commands
+
+A command is custom class that manages the logic between your input, usually a xref:develop:classes/forms.adoc[form] object, a object and an actor.
```

Comment:
I can't find what's an Actor in the Decidim terminology, maybe it's better to just call them Users? 

```suggestion
A command is custom class that manages the logic between your input, usually a xref:develop:classes/forms.adoc[form] object, an object and an user.
```